projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/salesforce-card.tsx

```diff
@@ -49,7 +60,7 @@ const SalesforceCard = ( {
 		isHeaderToggleEnabled: isValidSalesforceOrgId( salesforceData.organizationId ),
 		onHeaderToggleChange,
 		isConnected: isValidSalesforceOrgId( salesforceData.organizationId ),
-		isLoading: typeof data.isInstalled === 'undefined',
+		isLoading: ! data || typeof data.isInstalled === 'undefined',
```

Comment:
While updating this file, just updated this to check for presence of `data` like we're doing in other files.